src/handlers/registration.py

```
@@ -1,5 +1,5 @@
-"""
-Обработчики регистрации
+﻿"""
+РћР±СЂР°Р±РѕС‚С‡РёРєРё СЂРµРіРёСЃС‚СЂР°С†РёРё
```

Ruff (0.12.2)

Docstring contains ambiguous `Р` (CYRILLIC CAPITAL LETTER ER). Did you mean `P` (LATIN CAPITAL LETTER P)?

(RUF002)